frontend/src/components/Footer/Footer.scss

```
@@ -0,0 +1,56 @@
+.aha__footer {
+    margin-top: 30px;
+    padding-top: 20px;
+    padding-bottom: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    text-align: center;
+
+    p {
+        max-width: 700px;
+    }
```

I would apply this to every child element of the disclaimer and the privacy containers, or to the disclaimer and privacy containers themselves. And probably with centering them using margins as well:

```suggestion
    .disclaimer > * {
        max-width: 700px;
        margin: 0 auto;
    }
```